reference/docs-conceptual/learn/ps101/00-introduction.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Introduction | PowerShell 101
```

Comment:
Because of the pipe, I don't think the title on the tab is rendering the way you'd expect and this will look odd in Search.

Reply:
Removed it in all docs